about.html

```diff
@@ -0,0 +1,197 @@
+              <a href="index.html#sponsors" class="nav__links">Sponsor</a>
+            </li>
+            <li class="nav__list">
+              <a href="#speakers" class="nav__links">Speakers</a>
+            </li>
+            <li class="nav__list">
+              <a href="about.html#check" class="nav__links">Africa Rise</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="intro">
+      <div class="header__content">
+        <ul class="hoo">
+          <li><img src="images/home.png" alt="home" class="home"></li>
+          <li class="dash">&gt;</li>
+          <li>조선말</li>
+          <li class="dash">&gt;</li>
+          <li>CC조선말</li>
+      </ul>
+
+        <h2 class="heading-sub2">
+          "Lets Connect and Build Africa."
+        </h2>
+        <h1 class="heading-primary1">
+          AFRICA YOUTH SUMMIT 2022
+        </h1>
+        <p class="heading-theme1" id="aboutto">
+          Young people make up the bulk of Africa’s total population with an estimated 75% of the continent’s population below the age of 35. African Youth Summit is working with young leaders in all African countries and joining forces with civil society, companies and people on the frontline of climate change, education and skills improvement, health and well-being, good governance, human rights and accountability, employment opportunities and leadership skills to empower young leaders with the neccessary knowledge to develop and transform Africa into a better place.
+        </p>
+        <p class="reach">Feel free to contact us about the African Youth Sumit via our email or incase you have any questions.
+          <a href="africasummit@gmail.com" class="mail">africasummit@gmail.com</a>  
+          </p>  
+      </div>
+    </section>
+
+    <section class="creativelogo">
+      <h3 class="describ">Africa Youth Summit Logo</h3>
+      <p class="aboutLogo">Africa Youth Summit, shares the history of uplifiting young african youth into great leaders and influencers in the continent and the entire world.The semi-circlular white lines in the background colors in their logo the desire to cover the entire geographic area of Africa representing its true mission of training all African youth to be great leaders</p>
+      <img src="./images/africalogo.jpg" alt="image of africa" class="africano"/>
+  </section>
+
+  <section class="events">
+          <h3 class="pastmeso">Take a look at the past two Africa Youth Summit events which took place in Egypt and Rwanda</h3>
+          <div class="picspast">
+                  <div class="head">
+                      <h2 class="year">2020</h2>
+                      <p class="summit">Africa Youth Summit 2020 in Egypt(Cairo)</p>
+                  </div>
+                  <div class="head">
+                      <h2 class="year">2018</h2>
+                      <p class="summit">Africa Youth Summit 2020 in Rwanda(Kigali)</p>
+                  </div>
+          </div>
```

Comment:
- [ ] Could you please ensure that this text is aligned to the center, this little offset makes them look a little odd. You can use flex-box and its properties like `justify-content` and `align-items` to fix it
![image](https://user-images.githubusercontent.com/52635378/170831967-ee925fba-b86b-4341-9774-03de8c85f0a3.png)